internal/services/automation/automation_dsc_configuration_resource_test.go

```diff
@@ -61,7 +61,6 @@ func TestAccAutomationDscConfiguration_complete(t *testing.T) {
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
 		},
-		data.ImportStep(),
```

Comment:
we should be able to leave this one in?

```suggestion
		data.ImportStep(),
```